src/components/CopyModal.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import DialogContent from '@material-ui/core/DialogContent';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+
+const CopyModal = ({ token, setOpenCopy }) => {
+  function copy() {
```

Comment:
Very cool! I have never implemented/seen how to copy to a users clipboard.

Reply:
Cool right? https://www.w3schools.com/howto/howto_js_copy_clipboard.asp